src/utilities/utilities.js

```diff
@@ -216,6 +216,185 @@ export function body_surface_area(weight, height, formula = "dubois") {
     return 0.244 * Math.pow(weight, 0.383) * Math.pow(height, 0.693);
 
   throw new Error(
-    `This ${formula} to calculate the body_surface_area does not exists.`
+    `This ${formula} to calculate the body_surface_area does not exists.`,
   );
 }
+
+/**
+ * Estimates the relative air speed which combines the average air speed of the
+ * space plus the relative air speed caused by the body movement. Vag is assumed
+ * to be 0 for metabolic rates equal and lower than 1 met and otherwise equal to
+ * Vag = 0.3 (M - 1) (m/s)
+ *
+ * @template {(number | number[])} T
+ * @param {T} v - air spped measured by the sensor, [m/s]
+ * @param {number} met - metabolic rate, [met]
+ * @returns {T} relative air speed, [m/s]
+ */
+export function v_relative(v, met) {
+  if (met <= 1) return v;
+  if (Array.isArray(v)) {
+    return v.map((_v) => _v_relative_single(_v, met));
+  }
+  return _v_relative_single(v, met);
+}
+
+/**
+ * @param {number} v
+ * @param {number} met
+ * @returns {number}
+ */
+function _v_relative_single(v, met) {
```

Comment:
The name of this function is not very informative

Reply:
See my comment on `clo_dynamic_single` for the rationale around this decision.